Sources/MapboxMaps/Foundation/BaseMapView.swift

```diff
@@ -32,6 +30,22 @@ open class BaseMapView: UIView, CameraViewDelegate {
     /// List of completion blocks that need to be completed by the displayLink
     internal var pendingAnimatorCompletionBlocks: [PendingAnimationCompletion] = []
 
+    /// Pointer HashTable for holding camera animators
+    internal var cameraAnimatorsHashTable = NSHashTable<CameraAnimatorProtocol>.weakObjects()
+
+    /// List of animators currently alive
+    public var cameraAnimators: [CameraAnimator] {
```

Comment:
Do developers need to be able to see whether any non-CameraAnimator animators are animating?

Reply:
I think we should allow that for sure.. We'd need to introduce some new type to represent both `CameraAnimator` and `FlyToAnimator`.

Reply:
Rename `CameraAnimator` to `BasicCameraAnimator`
Rename `FlyToAnimator` to `FlyToCameraAnimator`
Rename `CameraAnimatorProtocol` to `CameraAnimator`
Update implementation of `cameraAnimators` to return all objects in the hash table.

Reply:
Change to `Set<CameraAnimator>` since we can't guarantee ordering

Reply:
This is done.